.github/pull_request_template.md

```diff
@@ -0,0 +1,11 @@
+_Describe your feature here_
```

Comment:
Hmm... this file should not be showing up in the diff. I think you may have forgotten to pull from main before opening this PR. (Please always do this to ensure your code is up to date and to avoid merge conflicts when you open future PRs)

Reply:
Hmm I think I got this from pulling from main and then merging main with my current branch. Not sure what went wrong here — I'll get rid of it from this commit.